Add site info on login page

marceloFA opened the pull request:
This adds an onto the login page a few pieces of information about cubers.io.
I assume this is completely up to changes and represents only an initial try on the issue.

![draft_info_on_login](https://user-images.githubusercontent.com/29216027/95237162-3ea3fa00-07de-11eb-89eb-3624e13bc251.png)

![mobile_preview](https://user-images.githubusercontent.com/29216027/95251511-0b6b6600-07f2-11eb-87a1-b313e0933464.png)

euphwes commented:
This looks great! Definitely what I had in mind, and it looks cleaner than what I probably would have put together myself. Couple of quick notes and questions:

- The blurb under the cubers.io logo, can you shorten it a bit, something like "Participate in WCA-style speedcubing competitions!"
- Under "Ease of Use", the "automatic Reddit comment generation and submission" point is outdated. This is from when we used to have a weekly post in Reddit, and participating in the comp here at the site would automatically post a comment with your results into the corresponding thread. Let's replace this with something like "Reddit integration provides weekly notifications of new competitions and statistics about your solves."
- Under "Weekly Competitions", can you slim down the bit about the timer? Something to the effect of "Use the built-in customizable timer, or use your own and enter results directly."
- "Leaderboards and Rankings" -> lowercase rankings
- How does this look on mobile? I sometimes accidentally neglect mobile users since I mostly use this on my PC.

euphwes commented:
Also, I know this is a draft, but whenever you submit the final pull request, can you tweak the commit message to have "[#121]" in front, and "Resolves #121." in the message somewhere? Since this PR Not a big deal, but it'll match the pattern I've tried to establish with marking commit messages to show which issue they correspond to, and the "resolves" bit will automatically close the issue once this is merged into the master branch.

Thanks for picking this up!

marceloFA commented:
Here are the updates:
- [x] The blurb has a shorter text
- [x] Updated Reddit integration feature explanation
- [x] Update timer feature explanation
- [X] Fix typo lowercasing 'Rankings'
- [X] Updated desktop preview and added mobile preview

Now one thing is still pending: The ``card-deck`` div on mobile overflows the main ``center-container`` div as the third image on the PR description shows. I'm not aware of how to fix this. I experimented with changing the ``display`` prop on both divs but it didn't fix the issue. Do you have any idea how to fix it?

euphwes commented:
I'm definitely not a UI expert (as you can probably tell by the horrible HTML and LESS/CSS, haha). I'll toy with a bit and see what I come up with.

euphwes commented:
Also I probably wasn't super clear: "[#121] Add site info on login page. Resolves #121" should be the commit message itself, for the final commit. You can keep the name of PR itself whatever you'd like. It's looking good! Gonna pull down your changes locally and toy with them a bit to see if I can figure out how to make mobile work a little nicer.

euphwes commented:
Just posted a small PR to this branch in your repo with another commit that fixes the mobile issue and tweaks some spacing. If you want to merge that with your commits and update the message and stick that into this PR, I think it's in good shape to accept!
